dm/syncer/syncer.go

```diff
@@ -19,8 +19,6 @@ import (
 	"crypto/tls"
 	"fmt"
 	"math"
-	"os"
```

Comment:
the file permission has been changed from 644 to 755 🤔 please revert it

Reply:
ping @WizardXiao

Reply:
sorroy, it may be some kind of misoperation, fortunately, it was found out when you review. @lance6716 @Ehco1996